tools/upgrade-prerequisites.sh

```diff
@@ -0,0 +1,48 @@
+#! /usr/bin/bash
+
+# This script is intended to be run in CI to test upgrades.
+# It executes any preparation steps that must be perfomed before upgrading
+# OpenStack services.
+
+# NOTE(upgrade): This script is unique to each release. It may not be required
+# for some releases.
+
+set -ex
+
+function prechecks() {
+    if [[ ! $KAYOBE_CONFIG_PATH ]]; then
+        echo "Environment variable \$KAYOBE_CONFIG_PATH is not defined"
+        echo "Ensure your environment is set up to run kayobe commands"
+        exit 2
+    fi
+}
+
+function rabbit_upgrade() {
+    # Ensure RabbitMQ is upgraded to 3.13
+    if kayobe overcloud host command run -l controllers -b --command "docker exec rabbitmq rabbitmqctl --version | grep -F 3.11" --show-output; then
```

Comment:
I think this should be a safe check, given RabbitMQ are moving on to versions 4.x now. But it is possible this could pick up when the two minor points of a version are 3.11. e.g: 3.13.11. 

Probably not an issue, but we could change the grep anyway?

```suggestion
    if kayobe overcloud host command run -l controllers -b --command "docker exec rabbitmq rabbitmqctl --version | grep -F 3.11." --show-output; then
```